bitrise.yml

```diff
@@ -1216,7 +1216,7 @@ workflows:
             # debug log
             set -x
             # get dependency
-            cd ./test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
+            cd ./test-fixtures && git clone https://github.com/clarmso/xcresult_extract.git
```

Comment:
I forked the repo just to demonstrate quickly how the `--legacy` flag would work. @isabelrios Please feel free to add this flag in your script.